src/protocol/dp/distributions.rs

```diff
@@ -0,0 +1,118 @@
+use rand::{
+    distributions::{Distribution, Uniform},
+    Rng,
+};
+
+use std::{f64::consts::PI, fmt::Debug};
+
+#[cfg(all(test, unit_test))]
+pub fn close(a: f64, b: f64, precision: u8) -> bool {
```

Comment:
A trait might be better, but I'm going to say at least name this `is_close()` or `is_approx_equal()`

Also, please document `precision`.  I understand that the goal is to test that the difference between values is less than `10^-precision`.